src/app/profile/page.tsx

```diff
@@ -1,11 +1,160 @@
-import Link from 'next/link';
+"use client"; // This line makes the file a client component
+
+import React, { useState, useEffect } from 'react';
+
+const EditableText: React.FC = () => {
+  // Initialize state for each editable text field
+  const savedText1 = localStorage.getItem('editableText1') || '';
+  const savedText2 = localStorage.getItem('editableText2') || '';
+  const savedText3 = localStorage.getItem('editableText3') || '';
+  const savedText4 = localStorage.getItem('editableText4') || '';
+  const savedText5 = localStorage.getItem('editableText5') || '';
+  const savedText6 = localStorage.getItem('editableText6') || '';
+  const savedText7 = localStorage.getItem('editableText7') || '';
+  const savedResume = localStorage.getItem('resumeFile') || '';
```

Comment:
The build failed because of these lines. I know you guys can't see the deployment on Vercel but if you run `npm run build` before you push it will build the project in production mode and you'll see the errors locally. This is a good habit to get into in order to catch weird errors and warnings sooner.

Reply:
This is what I'm seeing on Vercel. If you do `npm run build` it should show you something similar, then you can go ahead and fix the issues.

<img width="1192" alt="Screenshot 2025-04-01 at 3 15 52 PM" src="https://github.com/user-attachments/assets/db9150ba-9089-4ac6-8898-7e20de72f726" />